djvm/src/main/kotlin/net/corda/djvm/rewiring/SandboxClassLoader.kt

```diff
@@ -419,14 +419,13 @@ class SandboxClassLoader private constructor(
                 val resource = supportingClassLoader.getResource(resourceName)
                         ?: throw ClassNotFoundException("Class file not found: $resourceName")
 
-                if (externalCaching) {
+                if (externalCaching && externalCache != null) {
```

Comment:
Is it possible for `externalCache` to be `null` and `externalCaching` true?

Reply:
No, it's not. The `externalCache` property is cunningly defined as:
```kotlin
var externalCaching: Boolean = (externalCache != null)
    set(value) {
        field = value && (externalCache != null)
    }
```
:fox_face:

Reply:
Ah, I see :+1: